PolicyGenerator.go

```diff
@@ -69,12 +66,58 @@ type plugin struct {
 	outputBuffer bytes.Buffer
 }
 
-//nolint: golint
-//noinspection GoUnusedGlobalVariable
-var KustomizePlugin plugin
+func main() {
+	index := 2
+	if os.Args[0] == "./PolicyGenerator" {
+		index = 1
+	}
+	paths := os.Args[index:]
+	var outputBuffer bytes.Buffer
+
+	for _, path := range paths {
+		dir, err := os.ReadDir(path)
+		if err != nil {
+			p := plugin{}
+			file, err := os.ReadFile(path)
+			if err != nil {
+				panic(err)
+			} else {
+				err := p.Config(file)
+				if err != nil {
+					panic(err)
+				}
+				output, err := p.Generate()
+				if err != nil {
+					panic(err)
+				}
+				outputBuffer.Write(output)
+			}
+		} else {
+			for _, entry := range dir {
+				if !entry.IsDir() {
+					file, err := os.ReadFile(path + "/" + entry.Name())
```

Comment:
That way the path joining is OS agnostic.
```suggestion
					file, err := os.ReadFile(path.Join(path, entry.Name()))
```